packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/manageContacts/manage-contacts.controller.js

```diff
@@ -171,15 +163,44 @@ export default /* @ngInject */ function TelecomTelephonyBillingAccountManageCont
     });
   }
 
+  function getPackXdslServiceIds(services) {
+    const packIds = [];
+    return $q
+      .all(
+        services.map(({ serviceName }) => {
+          return $http
+            .get(`/pack/xdsl/search?pattern=${serviceName}`)
+            .then(() => {
+              packIds.push(serviceName);
+            })
+            .catch(() => {
+              // nothing to do
+            });
```

Comment:
If nothing to do, remove the catch?

Reply:
I don't have full context, but keeping the catch as is may allow to not reject the promise chain, which could break the page loading.